platforms/r3-corda-ent/charts/zone/values.yaml

```diff
@@ -0,0 +1,231 @@
+# Eg. nodeName: zone
+nodeName: 
+
+# Display more messages while executing Kubernetes deployment
+bashDebug: false
+# Provide prefix for deployment
+# Eg. prefix: cenm
+prefix:
+
+# This section contains the Corda Enterprise Zone metadata.
+metadata:
+  # Provide the namespace for the Corda Enterprise Zone.
+  # Eg. namespace: cenm
+  namespace:
+
+# Provide information regarding the Docker images used.
+image:
+  # Provide the alpine utils image, which is used for all init-containers of deployments/jobs.
+  # NOTE: The alpine image used is the base alpine image with CURL installed.
+  # Eg. initContainerName: hyperledgerlabs/alpine-utils:1.0
+  initContainerName:
+  # Provide the image for the main Zone container.
+  # Eg. zoneContainerName: corda/enterprise-zone:1.5.1-zulu-openjdk8u242
+  zoneContainerName:
+  # Pull policy to be used for the Docker image
+  # Eg. pullPolicy: Always
+  pullPolicy:
+
+#############################################################
+#                       Settings                            #
+#############################################################
+config:
+  # Provide volume related specifications
+  volume:
+    # Eg. baseDir: /opt/cenm
+    baseDir:
+podSecurityContext:
+  # Eg. runAsUser: 1000
+  runAsUser:
+  # Eg. runAsGroup: 1000
+  runAsGroup:
+  # Eg. fsGroup: 1000
+  fsGroup:
+
+#############################################################
+#                  CENM Service Details                     #
+#############################################################
+# This section details the CENM service names as per the configuration file
+# It also contains the passwords for keystores and truststores
+cenmServices:
+  # Provide the name of the idman
+  # Eg. idmanName: idman
+  idmanName:
+  # Provide the password of corda-ssl-trust-store.jks
+  # Eg. sslTruststore: password
+  sslTruststore:
+  # Provide the password of corda-ssl-identity-manager-keys.jks
+  # Eg. sslIdmanKeystore: password
+  sslIdmanKeystore:
+
+#############################################################
+#                Database Options and Configuration         #
+#############################################################
+# Database configuration
+database:
+  # Java class name to use for the database
+  # Eg. driverClassName: "org.h2.Driver"
+  driverClassName:
+  # JDBC Driver name
+  # Eg. jdbcDriver: ""
+  jdbcDriver:
+  # The DB connection URL 
+  # Eg. url: "jdbc:h2:file:./h2/zone-persistence;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=10000;WRITE_DELAY=0;AUTO_SERVER_PORT=0"
+  url:
+  # DB user
+  # Eg. user: "example-db-user"
+  user:
+  # DB password 
+  # Eg. password: "example-db-password"
+  password:
+  # Migrations of database can be run as part of the startup of Zone, if set to true. 
+  # If set to false, it will be run prior to setting up the Zone.
+  # Eg. runMigration: "true"
+  runMigration:
+
+# Volume size for h2/ directory
+# Eg. volumeSizeZoneH2: 1Gi
+volumeSizeZoneH2:
```

Comment:
define volumes in a single place, not multiple times

Reply:
fixed